### ISSUE TITLE
bug : build failed in RN 0.70

### ISSUE THREAD

**Issue by Productivix** [dashdoc/react-native-system-sounds#3]
in RN 0.70 : 
```
FAILURE: Build completed with 2 failures.

1: Task failed with an exception.
-----------
* Where:
Build file 'C:\myproject\node_modules\@dashdoc\react-native-system-sounds\android\build.gradle' line: 23

* What went wrong:
A problem occurred evaluating project ':dashdoc_react-native-system-sounds'.
> Plugin with id 'maven' not found.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
==============================================================================

2: Task failed with an exception.
-----------
* What went wrong:
A problem occurred configuring project ':dashdoc_react-native-system-sounds'.
> compileSdkVersion is not specified. Please add it to build.gradle

```

**Comment by shivsidan** [comment]
I am having the same issue - did you fix it? If Yes, please let me know

**Comment by Productivix** [comment]
yes somehow, as I had no reaction here , I integrated the package of  @gemedico/react-native-sounds 
hope will help you.

**Comment by solokhind** [comment]
Having the same issue on react native version 0.69.7

**Comment by asadwan** [comment]
Build failure is fixed in v1.0.1.

**Comment by solokhind** [comment]
It works now, thank you so much!
